backend/globaleaks/handlers/admin/langfiles.py

```diff
@@ -49,7 +48,6 @@ def post(self, lang):
         uploaded_file = self.get_file_upload()
         if uploaded_file is None:
             self.set_status(201)
-            self.finish()
             return
```

Comment:
Each of these requests is for uploading a chunk of the file and flow expects 200, 201 or 202 to say that the single chunk has been loaded correctly.

from https://github.com/flowjs/flow.js:

For every request, you can confirm reception in HTTP status codes (can be change through the permanentErrors option):

200, 201, 202: The chunk was accepted and correct. No need to re-upload.
404, 415. 500, 501: The file for which the chunk was uploaded is not supported, cancel the entire upload.
Anything else: Something went wrong, but try reuploading the file.